dev/com.ibm.websphere.appserver.features/visibility/public/mpRestClient-3.0/io.openliberty.mpRestClient-3.0.feature

```diff
@@ -22,11 +22,11 @@ Subsystem-Name: MicroProfile Rest Client 3.0
   io.openliberty.restfulWSClient-3.0, \
```

Comment:
```suggestion
  io.openliberty.restfulWSClient-3.0; ibm.tolerates:="3.1", \
```

Reply:
Adding toleration of 3.1 is good but won't work by itself. Using version 3.1 changes the version of resteasy being used and that then requires updates to the `io.openliberty.org.jboss.resteasy.mprestclient` bundle.

Reply:
Done

Reply:
Need a combination of Tom and Jim's changes to really test this with EE10.  The tolerates is needed so we pick the right private feature which is why I added that to the review.  Tom's changes will add more stuff.  So Jim you may need to make some updates for the mprestclient bundle if it fails to load due to 6.1.0 changes, but the hope is that it wouldn't, but until you test with EE10 features you won't know that.

Reply:
Hadn't seen Tom's comments prior to marking this as "done".